frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/sc-node-list.tsx

```diff
@@ -50,7 +50,7 @@ const getRows: GetRows = ({ componentProps, customData }) => {
     };
   });
 
-  if (!_.isEqual(filteredData, nodes)) {
+  if (setNodes && !_.isEqual(filteredData, nodes)) {
```

Comment:
Why is this required?

Reply:
Because its used here too - https://github.com/openshift/console/pull/6289/files#diff-99347a73d43d6e64cba499ad4c8093edR147 and setNodes is not required.